CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * [BUGFIX] Ingester: fixed deceiving error log "failed to update cached shipped blocks after shipper initialisation", occurring for each new tenant in the ingester. #1893
 * [BUGFIX] Ring: fix bug where instances may appear unhealthy in the hash ring web UI even though they are not. #1933
 * [BUGFIX] API: gzip is now enforced when identity encoding is explicitly rejected. #1864
+* [ENHANCEMENT] Upgrade Docker base images to `alpine:3.16.0`. #2028
```

Comment:
Don't hate me, please. As described in the contributing guide https://github.com/grafana/mimir/tree/main/docs/internal/contributing#changelog , the CHANGELOG entries are grouped togehter. This entry should be put at the end of the `[ENHNACEMENT]` entries group, right after line 28.